keras/backend/theano_backend.py

```diff
@@ -111,6 +111,13 @@ def dot(x, y):
     return T.dot(x, y)
 
 
+def batch_dot(x, y, axes=None):
+    if axes is None:
+        # behaves like tf.batch_matmul as default
```

Comment:
If we don't have a consistency between backends it would throw different errors in different backends, which I thought was not desired.